apps/ledger-live-desktop/tests/specs/speculos/ledgerSync.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { test } from "../../fixtures/common";
+import { AppInfos } from "@ledgerhq/live-common/e2e/enum/AppInfos";
+import { addTmsLink } from "tests/utils/allureUtils";
+import { getDescription } from "../../utils/customJsonReporter";
+import { commandCLI } from "tests/utils/cliUtils";
+
+const app: AppInfos = AppInfos.LS;
+
+test.describe(`[${app.name}] Sync Accounts`, () => {
+  const ledgerKeyRingProtocolArgs = {
+    getKeyRingTree: true,
+    pubKey: null,
+    privateKey: null,
+  };
+  const ledgerSyncPushDataArgs = {
+    pubKey: null,
+    privateKey: null,
+    rootId: null,
+    walletSyncEncryptionKey: null,
+    applicationPath: null,
+    push: true,
+    data: '{"accounts":[{"id":"mock:1:dogecoin:0.790010769447963:","currencyId":"dogecoin","index":1,"seedIdentifier":"mock","derivationMode":"","freshAddress":"1uVnrWAzycYqKUXSuNXt3XSjJ8"},{"id":"mock:1:bitcoin_gold:0.8027791663782486:","currencyId":"bitcoin_gold","index":1,"seedIdentifier":"mock","derivationMode":"","freshAddress":"1Y5T8JQqBKUS7cXbxUYCR4wg3YSbV9R"}],"accountNames":{"mock:1:dogecoin:0.790010769447963:":"Renamed Dogecoin 2","mock:1:bitcoin_gold:0.8027791663782486:":"Bitcoin Gold 2"}}',
```

Comment:
Why are we adding mock account ?
Next step would be to get these account with CLI and Speculos right ?

Reply:
I'm pushing mocks for now because it's only a V1 and I don't have the data for my own accounts 😬
Yes in the future the goal is to get the accounts with the CLI